File: Bridge/System/IO/FileNotFoundException.cs
using Bridge;
using System.Runtime.Serialization;
using System.Text;
using System.Runtime.InteropServices;

namespace System.IO
{

    public class FileNotFoundException : IOException
    {
        public FileNotFoundException()
            : base("Unable to find the specified file.")
        {
        }

        public FileNotFoundException(string message)
            : base(message)
        {
        }

        public FileNotFoundException(string message, Exception innerException)
            : base(message, innerException)
        {
        }

        public FileNotFoundException(string message, string fileName)
            : base(message)
        {
        }

        public FileNotFoundException(string message, string fileName, Exception innerException)
            : base(message, innerException)
        {
        }
        
        public string FileName
        {
            get { throw new NotImplementedException(); }
        }

        public string FusionLog
        {
            get { throw new NotImplementedException(); }
        }

        public override string Message
        {
            get
            {
                throw new NotImplementedException();
            }
        }
        
        public override string ToString()
        {
            throw new NotImplementedException();
        }
    }
}

File: Bridge/Resources/.generated/system/iO/fileNotFoundException.js
    Bridge.define("System.IO.FileNotFoundException", {
        inherits: [System.IO.IOException],
        props: {
            FileName: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            },
            FusionLog: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            },
            Message: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            }
        },
        ctors: {
            ctor: function () {
                this.$initialize();
                System.IO.IOException.$ctor1.call(this, "Unable to find the specified file.");
            },
            $ctor1: function (message) {
                this.$initialize();
                System.IO.IOException.$ctor1.call(this, message);
            },
            $ctor2: function (message, innerException) {
                this.$initialize();
                System.IO.IOException.$ctor2.call(this, message, innerException);
            },
            $ctor3: function (message, fileName) {
                this.$initialize();
                System.IO.IOException.$ctor1.call(this, message);
            },
            $ctor4: function (message, fileName, innerException) {
                this.$initialize();
                System.IO.IOException.$ctor2.call(this, message, innerException);
            }
        },
        methods: {
            toString: function () {
                throw new System.NotImplementedException();
            }
        }
    });
